custom_components/better_thermostat/models/models.py

```diff
@@ -78,9 +78,10 @@ def convert_outbound_states(self, hvac_mode):
 		if config.get('calibration_round'):
 			local_temperature_calibration = int(math.ceil(local_temperature_calibration))
 		if config.get('calibration_type') == 0:
-			current_heating_setpoint = state.get('current_heating_setpoint')
+			current_heating_setpoint = self._target_temp
 		elif config.get('calibration_type') == 1:
 			current_heating_setpoint = local_temperature_calibration
+			local_temperature_calibration = None
```

Comment:
Why is it important to set this to None?

Reply:
Because we use the target temp for calibration, not the local temperature calibration setting of the TRV

Reply:
so you are changing. target temp every few minutes ? like the local logic of TRV does ? My manual observations are its best to keep the external sensor temp in sync with (trv temp - local_temp_calibration) then the TRV algorithm on the actual head does his job well

Reply:
No, its the target temp, not the internal, its for TRVs that havent the option to set a calibration

Reply:
> Because we use the target temp for calibration, not the local temperature calibration setting of the TRV

Okay, but why do we want to set it to None here? 

We don't use the value.

Reply:
+1 Its a different Topic, the local_calibration doesn't turn off the internal algorithm anyway

Reply:
you guys realize you just wasted 2 comments 2 notifications about me going offtopic while that offtopic was resolved quickly by KartoffelToby right :) ?

Reply:
@gacopl well, I wasn't sure that your question was completely answered. So I was pointing to the discussion tab for further question about that topic :)